internal/provider/models/processors/test/aggregate_v2_test.go

```diff
@@ -223,127 +223,99 @@ func TestAggregateV2Processor(t *testing.T) {
-			},
-
-			// Error: server-side validation - invalid window duration
-			{
-				Config: GetCachedConfig(cacheKey) + `
-				resource "mezmo_aggregate_v2_processor" "my_processor" {
-					pipeline_id 	= mezmo_pipeline.test_parent.id
-					inputs 			= [mezmo_http_source.my_source.id]
-					method 			= "sliding"
-					strategy 		= "average"
-					window_duration = 305325235325
-				}`,
-				ExpectError: regexp.MustCompile("/window_duration/maximum"),
-			},
-
-			// Error: window duration
-			{
-				Config: GetCachedConfig(cacheKey) + `
-					resource "mezmo_aggregate_v2_processor" "my_processor" {
-						pipeline_id 	= mezmo_pipeline.test_parent.id
-						inputs 			= [mezmo_http_source.my_source.id]
-						strategy 		= "sum"
-						method 			= "tumbling"
-						interval 		= 10
-						window_duration = 5
-					}`,
-				ExpectError: regexp.MustCompile("The field 'window_duration' can only be set if method == 'sliding'"),
-			},
-
-			// Error: window min
-			{
-				Config: GetCachedConfig(cacheKey) + `
-					resource "mezmo_aggregate_v2_processor" "my_processor" {
-						pipeline_id = mezmo_pipeline.test_parent.id
-						inputs 		= [mezmo_http_source.my_source.id]
-						strategy 	= "sum"
-						method 		= "tumbling"
-						interval 	= 10
-						window_min 	= 5
-					}`,
-				ExpectError: regexp.MustCompile("The field 'window_min' can only be set if method == 'sliding'"),
-			},
-
-			// Error: interval
-			{
-				Config: GetCachedConfig(cacheKey) + `
-					resource "mezmo_aggregate_v2_processor" "my_processor" {
-						pipeline_id 	= mezmo_pipeline.test_parent.id
-						inputs 			= [mezmo_http_source.my_source.id]
-						strategy 		= "sum"
-						method 			= "sliding"
-						interval 		= 10
-						window_duration = 5
-					}`,
-				ExpectError: regexp.MustCompile("The field 'interval' can only be set if method == 'tumbling'"),
-			},
-
-			// confirm manually deleted resources are recreated
-			{
-				Config: GetProviderConfig() + `
-				resource "mezmo_pipeline" "test_parent2" {
-					title = "pipeline"
-				}
-				resource "mezmo_aggregate_v2_processor" "test_processor" {
-					pipeline_id = mezmo_pipeline.test_parent2.id
-					title 		= "new title"
-					inputs 		= []
-					method      = "tumbling"
-  					interval    = 3600
-					strategy    = "sum"
-				}`,
-				Check: resource.ComposeAggregateTestCheckFunc(
-					resource.TestMatchResourceAttr(
-						"mezmo_aggregate_v2_processor.test_processor", "id", regexp.MustCompile(`[\w-]{36}`)),
-					resource.TestCheckResourceAttr("mezmo_aggregate_v2_processor.test_processor", "title", "new title"),
-					// delete the resource
-					TestDeletePipelineNodeManually(
-						"mezmo_pipeline.test_parent2",
-						"mezmo_aggregate_v2_processor.test_processor",
-					),
-				),
-				// verify resource will be re-created after refresh
-				ExpectNonEmptyPlan: true,
-			},
-
-			// Delete testing automatically occurs in TestCase
+			//	// Error: server-side validation for missing strategy
```

Comment:
can this be uncommented or removed?

Reply:
Yup, I'll do that. These are now being validated by the schema change so these test scenarios can no longer make it to our code.